test/framework/easystacks/test_easystack_easyconfigs_opts.yaml

```diff
@@ -0,0 +1,11 @@
+easyconfigs:
+    - binutils-2.25-GCCcore-4.9.3:
+        options: {
+          'debug': True,
```

Comment:
A test with more than one option set might be useful...and ultimately also one with an invalid setting!

Reply:
I'll extend this a bit so it uses multiple options, but handling of invalid options should be checked in #4057

Reply:
done in 0550fc0